kubeaudit.go

```diff
@@ -154,6 +156,18 @@ func (a *Kubeaudit) AuditManifest(manifest io.Reader) (*Report, error) {
 		return nil, err
 	}
 
+	for _, result := range results {
+		auditResults := result.GetAuditResults()
+
+		if !filepath.IsAbs(manifestPath) {
+			manifestPath = strings.TrimPrefix(filepath.Clean("/"+manifestPath), "/")
```

Comment:
check for absolute path.  users might pass in a relative path when calling kubeaudit (e.g `../../myfile`). when we add this path to the sarif, when uploading it to code scanning interfaces, such as Github Code Scanning, we cannot correctly display the preview